docs/federation/relayable.md

```diff
@@ -11,6 +11,9 @@ Such entities are:
 * [Like][like]
 * [PollParticipation][poll_participation]
 
+If the parent is a relayable itself, the parent of that should be used, until it isn't a relayable anymore.
```

Comment:
I feel that explanations in this section may be confusing.

Maybe we should change the text at lines 5-6 to use the word `root` instead of the word `parent`? And then we can explain what the `root` word means: it is a top parent for entity, which is normally a `Post` for both `Like` for a `Post` and `Like` for a `Comment`.

Also I guess we should replace `parent` with `root` everywhere in this document where we actually mean `root`. 

E.g. line 33: `The parent author then must envelop it in a new [Magic Envelope][magicsig] and send the entity to all the recipients of the parent entity.`. Here we actually mean `root`, becase for `Comment` for `Like` it is not the parent author, but the root author who should perform the relaying.

Reply:
Sure, you're right, I fixed that.